base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -166,6 +170,10 @@ public void update(VideoStreamingParameters params) {
             if (params.format != null) {
                 this.format = params.format;
             }
+
```

Comment:
Should we also add an update for preferredDiagonalScreenSize if it is available to this method?
```
if (!params.preferredDiagonal != null) {
    this.preferredDiagonal = params.preferredDiagonal
}
```

Reply:
@JulianKast I've modified the code a little bit to match with the primitive type